inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/project/RecommenderEditorPanel.html

```diff
@@ -18,111 +18,122 @@
-            </label>
-            <div class="col-sm-9">
-              <select wicket:id="feature" class="form-control" data-container="body"></select>
-            </div>
-          </div>
-          <div class="form-group" wicket:enclosure="tool">
-            <label wicket:for="tool" class="col-sm-3 control-label">
-              <wicket:label key="tool"/>
-            </label>
-            <div class="col-sm-9">
-              <select wicket:id="tool" class="form-control" data-container="body"></select>
-            </div>
-          </div>
-          <div class="form-group" wicket:id="activationContainer">
-            <label class="col-sm-3 control-label">
-              <wicket:message key="activation"/>
-            </label>
-            <div class="col-sm-9">
-              <div class="form-group" wicket:enclosure="threshold">
-                <label wicket:for="threshold" class="col-sm-3 control-label">
-                  <wicket:label key="threshold"/>
-                </label>
-                <div class="col-sm-9">
-                  <input type="number" wicket:id="threshold" class="form-control"/>
-                </div>
-              </div>
-              <div class="form-group" wicket:enclosure="alwaysSelected">
-                <div class="col-sm-offset-3 col-sm-9">
-                  <div class="checkbox">
-                    <label wicket:for="alwaysSelected">
-                      <input wicket:id="alwaysSelected" type="checkbox">
-                      <wicket:label key="alwaysSelected"/>
-                    </label>
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
-          <div class="form-group" wicket:enclosure="maxRecommendations">
-            <label wicket:for="maxRecommendations" class="col-sm-3 control-label">
-              <wicket:label key="maxRecommendations"/>
-            </label>
-            <div class="col-sm-9">
-              <input type="number" wicket:id="maxRecommendations" class="form-control"/>
-            </div>
-          </div>
-          <div class="form-group" wicket:enclosure="statesForTraining">
-            <label wicket:for="statesForTraining" class="col-sm-3 control-label">
-              <wicket:label key="statesForTraining"/>
-            </label>
-            <div class="col-sm-9" wicket:id="statesForTraining"></div>
-          </div>
-          <div wicket:id="traitsContainer">
-            <div wicket:id="traits"></div>
-          </div>
-        </div>
-      </div>
-      <div class="panel-footer text-right">
-        <input type="button" wicket:id="delete" class="btn btn-danger pull-left" wicket:message="value:delete"></input>
-        <input type="submit" wicket:id="save" class="btn btn-primary" wicket:message="value:save"></input>
-        <input type="button" wicket:id="cancel" class="btn btn-default" wicket:message="value:cancel"></input>
-      </div>            
-    </form>
-  </wicket:panel>
+	<wicket:panel>
```

Comment:
This file contains lots of whitespace changes that need to be reverted. Please note that HTML files are indented using spaces only and using 2 spaces for an indentation.